pit30m/data/log_reader.py

```diff
@@ -52,6 +53,53 @@ class LiDARFrame:
 
 PARTITIONS_BASEPATH = "s3://pit30m/partitions/"
 
+# Original dtype for unified raw pose arrays. Regular users should be using specialized getters, such as those for
+# continuous or map-relative poses, not the raw poses. '?' entries are bool but this does not register for some arcane
+# reason.
```

Comment:
What do you mean by "this does not register"?

Reply:
Oh as in numpy doesn't "see" those fields as bool when you load the npy file and print the dtype. I clarified the comment.